client/my-sites/sidebar-unified/item.jsx

```diff
@@ -36,6 +38,9 @@ export const MySitesSidebarUnifiedItem = ( { title, icon, url, path, slug } ) =>
 	if ( isStats && selectedSiteId ) {
 		children = <StatsSparkline className="sidebar-unified__sparkline" siteId={ selectedSiteId } />;
 	}
+	url = config.isEnabled( 'nav-unification' )
+		? addQueryArgs( { flags: 'nav-unification' }, url )
+		: url;
```

Comment:
Transforming the data itself might be better handled in a central location. I have a PR for this here

https://github.com/Automattic/wp-calypso/pull/45992

Reply:
Either way, I feel this should be in another PR not this one.

Reply:
Couldn't test properly and thought it was temporary. You are right though. Removed https://github.com/Automattic/wp-calypso/pull/45829/commits/db189ea3bcdd303dbba68636ab396bac94ef2591